src/components/WebPlatformPlayer/InfoPanel.js

```diff
@@ -0,0 +1,157 @@
+// @flow
+import React, { useCallback } from "react";
+
+import { Trans } from "react-i18next";
+import { useSelector } from "react-redux";
+import { useTheme } from "@react-navigation/native";
+import { StyleSheet, View, TouchableOpacity, Linking } from "react-native";
+import { translateContent } from "@ledgerhq/live-common/lib/platform/logic";
+import type { TranslatableString } from "@ledgerhq/live-common/lib/platform/types";
+
+import { languageSelector } from "../../reducers/settings";
+import ExternalLinkIcon from "../../icons/ExternalLink";
+import AppIcon from "../../screens/Platform/AppIcon";
+import CloseIcon from "../../icons/Close";
+import BottomModal from "../BottomModal";
+import LText from "../LText";
+
+type Props = {
+  name: string,
+  icon?: string | null,
+  description: TranslatableString,
+  url?: string | null,
+  isOpened: boolean,
+  setIsOpened: (isOpened: boolean) => void,
+};
+
+const InfoPanel = ({
+  name,
+  icon,
+  description,
+  url,
+  isOpened,
+  setIsOpened,
+}: Props) => {
+  const settingsLocale = useSelector(languageSelector);
+  const { colors } = useTheme();
+
+  const onClosePress = useCallback(() => {
+    setIsOpened(false);
+  }, [setIsOpened]);
+  const onLinkPress = useCallback(url => {
+    Linking.openURL(url);
+  }, []);
+
+  return (
+    <BottomModal
+      style={{
+        ...styles.root,
+        backgroundColor: colors.card,
+      }}
+      id="InfoPanelModal"
+      isOpened={isOpened}
```

Comment:
You could add a `onClose` prop to the modal in order to close it when touching outside to be consistent with other bottom modals used on the app (i.e the transfer modal for exemple)

```suggestion
      isOpened={isOpened}
      onClose={onClosePress}
```

Reply:
Done 👍